packages/plantae/src/axios/createAxiosInterceptors.spec.ts

```diff
@@ -0,0 +1,517 @@
+import Axios from "axios";
+import { http } from "msw";
+import { describe, expect, it } from "vitest";
+
+import { base, baseURL, Status } from "../test/utils";
+import { server } from "../test/worker";
+import createAxiosInterceptors from "./createAxiosInterceptors";
+
+describe("createAxiosInterceptors", () => {
+  it("can modify request body", async () => {
+    server.use(
+      http.post(
+        base("/"),
+        async ({ request }) =>
+          new Response(null, {
+            status:
+              (await request.text()) === "modified" ? Status.OK : Status.BAD,
+          })
+      )
+    );
+
+    const axios = Axios.create({
+      baseURL,
+    });
+
+    const { request } = createAxiosInterceptors({
+      client: axios,
+      plugins: [
+        {
+          name: "plugin-modify-request-body",
+          hooks: {
+            beforeRequest: (req) => {
+              return new Request(req, {
+                body: "modified",
+                method: "POST",
+              });
+            },
+          },
+        },
+      ],
+    });
+
+    axios.interceptors.request.use(request.onFulfilled, request.onRejected);
+
+    const res = await axios.post("/");
+
+    expect(res.status).toBe(Status.OK);
```

Comment:
p2

I would like to know why we must recognize 17 line, it would be just a preparation level for tests IMHO. 

API could not know  whether request is modified in real world.

I suggest that it should confirm request text instead of status code.

Reply:
Accepted:
Body was used instead of status code for success flags.